src/Schema/Values/OwnerType.ts

```diff
@@ -97,6 +99,7 @@ export type PageOwnerType =
   | OwnerType.gene
   | OwnerType.home
   | OwnerType.partner
+  | OwnerType.profile
```

Comment:
We may not need this in `PageOwnerType` unless there is a corresponding Force `profile`.

Reply:
Yeah, I'm not sure about what happens with web's equivalent of iOS's profile screen and sub-screens. For example at `https://www.artsy.net/profile/edit`: 

![Screen Shot 2020-11-05 at 6 28 06 PM](https://user-images.githubusercontent.com/26879583/98307816-b3f91b00-1f94-11eb-9fe2-a215df5a6f8d.png)

Reply:
I can remove it for now, if that's preferable.